go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -70,21 +74,64 @@ type serviceConfig struct {
 	service *kapi.Service
 	// Were those rules etp:local + Host Networked rules
 	etpHostRules bool
+	// Used when svc=etp:local + it has non local host networked backends in LGW.
+	// NOTE: For most part this is always the opposite of etpHostRules, however there
+	// can be a situation where there are no endpoints for a service in which case
+	// both etpHostRules & etpNonHostRules will be false.
+	etpNonHostRules bool
```

Comment:
would it be better to call this etpLocalNoHostEndpoints (and refactor line 76 etpHostRules)?

Reply:
or does this really mean etpLocalAndOVNLocalEndpoints?

Reply:
yes then we'll also have to change the variable name before this... as in...

etpHostRules - needs to be etpLocalNodeHostEndpointsRules &
etpNonHostRules - will be etpNonLocalNodeHostEndpointsRules